legal-api/report-templates/template-parts/business-summary/amalgamations.html

```diff
@@ -1,12 +1,12 @@
 {% if business.amalgamatedEntity %}
 <div class="no-page-break">
   <div class="separator mt-4"></div>
-  <div class="section-title mt-4">Amalgamating Corporation(s) Information</div>
+  <div class="section-title mt-4">Amalgamating Businesses Information</div>
```

Comment:
Since was written as something can be singular or plural, should it be Business(es)?

Reply:
I thought of that, but amalgamations will always have a minimum of two businesses since we are combining them.

Reply:
Will confirm with Yui just in case but that was what is stated here:
![image](https://github.com/bcgov/lear/assets/144158015/1b179131-3364-4578-99e3-0a76ec58cc80)